network-proto/node.proto

```diff
@@ -121,4 +124,6 @@ service Node {
     rpc RecordTransaction (RecordTransactionRequest) returns (RecordTransactionResponse);
```

Comment:
While we are at it, we can remove `ProposeTransactions` and `RecordTransaction` as these are replaced by the new messages.